### DISCUSSION TITLE
How to toggle CSS with conditional

### DISCUSSION THREAD

**Discussion by CosmanAgraz** [NativeScript/NativeScript#9863] [Q&A] [answered]
I'm trying to implement a module very similar to the one shown [here](https://play.nativescript.org/?template=play-tsc&id=uQmH4X&v=1)

I have a list of items, and multiple items can be selected.  However, when I add a conditional to a <Label> class prop, the CSS does not update, only after a page refresh.

I managed to fix this issue by forcing a page refresh, but performance slowed waaaayyy down and I'm just wondering why is it that Image src prop can be updated without needing to refresh but not the class prop.

**Comment by rigor789** [comment, marked as answer]
If you could share a bit of your code, would help us understand what's going on. Class bindings should generally work in lists.

**Reply by CosmanAgraz** [reply]
@rigor789  thanks for the response!  I fixed this issue by updating the `.class` property of the View (the XML element) in the View-Model function which is executed when a button is tapped.
